docs/faq.md

```diff
@@ -155,10 +155,10 @@
 !!!tip ""
     This is an issue primarily affecting macOS users, but may occur on other devices as well. If you are able to launch the homebrew menu, but you are not seeing some or any of your homebrew, you will need to unset the archive bit with Hekate.
 
-    1. Send the Hekate payload to your Switch by following these instructions: [here](https://nh-server.github.io/switch-guide/user_guide/emummc/sending_payload/)
+    1. Send the Hekate payload to your Switch by following these instructions: [here](https://nh-server.github.io/switch-guide/user_guide/emuMMC/sending_payload/)
     2. Click on `Tools` on the top menu bar
-    3. In the bottom left corner click on `Archive bit * AutoRCM`
+    3. In the bottom left corner click on `Archive bit * autoRCM`
     4. Click on `Fix Archive bit` - this might take a while
     5. Press `Close` in the top right corner
     6. Press `Home` in the top menu bar to get back from where you started
-    7. Follow the instructions at the Launching CFW page [here](https://nh-server.github.io/switch-guide/user_guide/emummc/launching_cfw/) to launch Atmosphere.
+    7. Follow the instructions at the Launching CFW page [here](https://nh-server.github.io/switch-guide/user_guide/emuMMC/launching_cfw/) to launch Atmosphere.
```

Comment:
Change the link from "https://nh-server.github.io/switch-guide/user_guide/emuMMC/launching_cfw/" to "user_guide/emummc/launching_cfw.md"